<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParkingLot</class>
 <widget class="QDialog" name="ParkingLot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parking Lot</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>45</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(238, 238, 220);
background-color: rgb(104, 20, 157);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="parking_label">
           <property name="font">
            <font>
             <family>Segoe UI Light</family>
             <pointsize>15</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Parking Lot # 1 (Vacant)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="Vacant" native="true">
        <layout class="QFormLayout" name="formLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <family>Segoe UI Light</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Rate / HR: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="rate">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border : 1 solid gray;
border-radius : 10;</string>
           </property>
           <property name="text">
            <string>50</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Segoe UI Light</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Payment : </string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="payment">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border : 1 solid gray;
border-radius : 10;</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Segoe UI Light</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Client</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboBox">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border : 1 solid gray;
border-radius : 10;</string>
           </property>
           <item>
            <property name="text">
             <string>Select --</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="Occupied" native="true">
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Segoe UI Light</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Balance : </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="balance">
           <property name="font">
            <font>
             <family>Segoe UI Light</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0php</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Segoe UI Light</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Overdue : </string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="overdue">
           <property name="font">
            <font>
             <family>Segoe UI Light</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0php</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Segoe UI Light</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Client : </string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="clientname">
           <property name="font">
            <font>
             <family>Segoe UI Light</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Andrew Cortez</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>Segoe UI Light</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Time Started: </string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="timestarted">
           <property name="font">
            <font>
             <family>Segoe UI Light</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>00:00:00</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="VacantButtons" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QWidget" name="widget_5" native="true"/>
         </item>
         <item>
          <widget class="QPushButton" name="save_button">
           <property name="minimumSize">
            <size>
             <width>71</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>71</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(238, 238, 220);
background-color: rgb(75, 148, 78);

border-radius : 0;
</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancel_button">
           <property name="minimumSize">
            <size>
             <width>71</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>71</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(196, 63, 29);
color: rgb(255, 255, 255);
border-radius:0

</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="OccupiedButtons" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QWidget" name="widget_7" native="true"/>
         </item>
         <item>
          <widget class="QPushButton" name="release_button">
           <property name="minimumSize">
            <size>
             <width>71</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>71</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(238, 238, 220);
background-color: rgb(75, 148, 78);

border-radius : 0;
</string>
           </property>
           <property name="text">
            <string>Release</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
